packages/mobile/src/components/chat-actions-drawer/ChatActionsDrawer.tsx

```diff
@@ -0,0 +1,82 @@
+import { View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { Text } from 'app/components/core'
+import { NativeDrawer } from 'app/components/drawer'
+import { useNavigation } from 'app/hooks/useNavigation'
+import type { AppState } from 'app/store'
+import { getData } from 'app/store/drawers/selectors'
+import { setVisibility } from 'app/store/drawers/slice'
+import { makeStyles } from 'app/styles'
+
+const CHAT_ACTIONS_MODAL_NAME = 'ChatActions'
+
+const messages = {
+  visitProfile: 'Visit Profile',
+  blockMessages: 'Block Messages'
+}
+
+const useStyles = makeStyles(({ spacing, typography, palette }) => ({
+  drawer: {
+    marginVertical: spacing(7),
+    alignItems: 'center'
+  },
+  text: {
+    fontSize: 21,
+    lineHeight: spacing(6.5),
+    letterSpacing: 0.233333,
```

Comment:
Huh. Saw this matches figma, but just noting that this seems odd... 🤔

Flagging for @piazzatron these are the kinda weird styles that make me raise an eyebrow and pray they don't render funny